src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -137,28 +133,242 @@ public static String getProgressBar(Track track) {
+        } catch (IllegalArgumentException e) {
+            MessageUtils.sendErrorMessage("The duration is not in the correct format! Try something like `1d`",
+                    channel);
+            return null;
+        }
     }
 
-    private static char getPrefix(GuildWrapper guild) {
-        return guild == null ? Constants.COMMAND_CHAR : guild.getPrefix();
+    /**
+     * Gets the changes between two lists.
+     *
+     * @param oldList The old lis
+     * @param newList The new List
+     * @return A map where the added objects are true, and the removed objects are false. the values the same are not included.
+     */
+    public static <T> Map<Boolean, List<T>> getChanged(List<T> oldList, List<T> newList) {
+        Map<Boolean, List<T>> changes = new HashMap<>();
+        List<T> removed = new ArrayList<>();
+        List<T> added = new ArrayList<>();
+        for (T oldItem : oldList) {
+            if (!newList.contains(oldItem)) {
+                removed.add(oldItem);
+            }
+        }
+        for (T newItem : newList) {
+            if (!oldList.contains(newItem)) {
+                added.add(newItem);
+            }
+        }
+        changes.put(true, added);
+        changes.put(false, removed);
+        return changes;
     }
 
-    public static String formatCommandPrefix(GuildWrapper guild, String usage) {
-        String prefix = String.valueOf(getPrefix(guild));
-        if (usage.contains("{%}"))
-            return usage.replaceAll("\\{%}", prefix);
-        return usage;
+    /**
+     * Gets the suffix the a day in a month
+     *
+     * @param n The day in the month to get a suffix.
+     * @return The suffix for they day.
+     */
+    public static String getDayOfMonthSuffix(final int n) {
+        if (n < 1 || n > 31) throw new IllegalArgumentException("illegal day of month: " + n);
+        if (n >= 11 && n <= 13) {
+            return "th";
+        }
+        switch (n % 10) {
+            case 1:
+                return "st";
+            case 2:
+                return "nd";
+            case 3:
+                return "rd";
+            default:
+                return "th";
+        }
     }
 
+    /**
+     * Message fields:
+     * - Message ID
+     * - Author ID
+     * - Channel ID
+     * - Guild ID
+     * - Raw Content
+     * - Timestamp (Epoch seconds)
+     *
+     * @param message The message to serialise
+     * @return The serialised message
+     */
+    public static String getRedisMessageJson(Message message) {
+        return FlareBot.GSON.toJson(new RedisMessage(
+                message.getId(),
+                message.getAuthor().getId(),
+                message.getChannel().getId(),
+                message.getGuild().getId(),
+                message.getContentRaw(),
+                message.getCreationTime().toInstant().toEpochMilli()
+        ));
+    }
+
+    /**
+     * Gets the String representing the verification level.
```

Comment:
Maybe mention we will return the actual characters of the HIGH and VERY_HIGH levels not that string itself.